Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+    owl:sameAs <http://example.com/LL/lagin.ttl#Conchobuir>.
+
+<#Nessa>
+    a foaf:Person;
+    irishRel:nomName "Nessa";
+    irishRel:genName "Nessa";
+    owl:sameAs <http://example.com/Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.ttl#Nessa>.
+
+<#FindFili-53f7b450>
+    a foaf:Person;
+    irishRel:nomName "Find Fili";
+    rel:childOf <#RusRuad>;
+    owl:sameAs <#Find>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#FindFile>.
+
+<#Ailill>
+    a foaf:Person;
+    irishRel:nomName "Ailill";
+    rel:childOf <#RusRuad>;
+    rel:childOf <#Máta>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#Ailill>.
+
+<#Máta>
+    a foaf:Person;
+    irishRel:genName "Máta";
+    foaf:gender "female";
+    owl:sameAs <http://example.com/LL/lagin.ttl#Mata>.
+
+<#CairpreNiaFer>
+    a foaf:Person;
+    irishRel:nomName "Cairpre Nia Fer";
+    rel:childOf <#RusRuad>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#CoirpreNiaFer>.
+
+<#Ailill-b3f0ae10>
+    a foaf:Person;
+    irishRel:nomName "Ailill";
+    owl:sameAs <#Ailill>;
+    rel:childOf <#Rossa-f8cdec00>;
+    rel:childOf <#MátaMuirisc>;
+    rel:spouseOf <#Medb>.
+
+<#Rossa-f8cdec00>
+    a foaf:Person;
+    irishRel:genName "Rossa";
+    irishRel:nomName "Rus";
+    owl:sameAs <#RusRuad>.
+
+<#MátaMuirisc>
+    a foaf:Person;
+    irishRel:genName "Máta Muirisc";
+    irishRel:nomName "Máta Muirisc";
+    rdfs:comment "do Feraib Ool n-Écmacht.";
+    owl:sameAs <#Máta>.
+
+<#Medb>
+    a foaf:Person;
+    irishRel:nomName "Medb";
+    rel:childOf <#EchachFeidlich>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#MedbChruachna>.
+
+<#EchachFeidlich>
+    a foaf:Person;
+    irishRel:genName "Echach Feidlich";
+    irishRel:nomName "Eochu Feidlich";
+    owl:sameAs <http://example.com/LL/lagin.ttl#EchachFeidlig>.
+
+<#Conchobuir>
+    a foaf:Person;
+    irishRel:genName "Conchobuir";
+    irishRel:nomName "Conchobar";
+    rel:antagonistOf <#Ailill-b3f0ae10>;
+    rel:antagonistOf <#Medb>;
+    owl:sameAs <#Conchobar>.
+
+<#Corpre>
+    a foaf:Person;
+    irishRel:nomName "Corpre";
+    rdfs:comment "h-i Temair";
+    owl:sameAs <#CairpreNiaFer>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Tr%C3%AD_meic_Ruaid>.
+
+<#Find>
+    a foaf:Person;
+    irishRel:nomName "Find";
+    rdfs:comment "i n-Alind";
+    owl:sameAs <#FindFili-53f7b450>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Tr%C3%AD_meic_Ruaid>.
+
+<#Ailill>
+    a foaf:Person;
+    irishRel:nomName "Ailill";
+    rdfs:comment "h-i Cruachain";
+    owl:sameAs <#Ailill-b3f0ae10>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Tr%C3%AD_meic_Ruaid>.
+
+<#Senchán>
+    a foaf:Person;
+    irishRel:nomName "Senchán";
+    owl:sameAs <#SenchánTorpéist>.
+
+<http://www.vanhamel.nl/codecs/Tr%C3%AD_meic_Ruaid>
+    prov:wasAttributedTo <#Senchán>.
+
+<#Orthanach-f7fe6a30>
+    a foaf:Person;
+    irishRel:nomName "Orthanach";
+    owl:sameAs <#Orthanach>;
+    rdfs:comment "[author of 'Can trí macco Ruaid din rind']".
```

Comment:
It is in the text ("ut Orthanach dixit"). Will add URL for poem and rearrange, as per our discussion.